eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxBody.hs

```diff
@@ -575,19 +578,19 @@ deriving via
 
 pattern AlonzoTxBody ::
   EraTxBody era =>
-  Set (TxIn (Crypto era)) ->
-  Set (TxIn (Crypto era)) ->
-  StrictSeq (AlonzoTxOut era) ->
-  StrictSeq (DCert (Crypto era)) ->
-  Wdrl (Crypto era) ->
-  Coin ->
-  ValidityInterval ->
-  StrictMaybe (Update era) ->
-  Set (KeyHash 'Witness (Crypto era)) ->
-  MultiAsset (Crypto era) ->
-  StrictMaybe (ScriptIntegrityHash (Crypto era)) ->
-  StrictMaybe (AuxiliaryDataHash (Crypto era)) ->
-  StrictMaybe Network ->
+  Set (TxIn (Crypto era)) -> -- 0
```

Comment:
maybe we could make this clearer with something like
```
  Set (TxIn (Crypto era)) -> -- key 0 in the cbor maj
```

?

Reply:
I was thinking of deleting those once I'm done, but I can also leave them in for future reference.

Reply:
makes sense, deleting is probably better.